config/core/test-configs.yaml

```diff
@@ -2375,6 +2383,7 @@ test_configs:
   - device_type: qemu_arm64-virt-gicv3
     test_plans:
       - baseline_qemu
+      - kselftest-arm64
```

Comment:
There's no support yet for running kselftest on QEMU in KernelCI, so please drop this for now.